### ISSUE TITLE
WIP:  Build Eclipse OpenJ9 on Windows

### ISSUE THREAD

**Issue by mleipe** [eclipse/openj9#55]
Much of the work to be done to get Eclipse OpenJ9 build on Windows will be done in the https://github.com/ibmruntimes/openj9-openjdk-jdk9/ repository, as the makefiles, configure scripts, etc. live there.

I'm opening this issue: 1) to provide visibility of the work that's (mostly) happening in the other repository, and 2) to cover any work that needs to be done here.

@vsebe is working on this now.

**Comment by mleipe** [comment]
Here's the issue in the other repo:  https://github.com/ibmruntimes/openj9-openjdk-jdk9/issues/18

**Comment by mstoodle** [comment]
@vsebe could you write a short update on your progress here? I've had numerous inquiries about Windows support :)

**Comment by vsebe** [comment]
Starting with [Windows SDK 8](https://developer.microsoft.com/en-us/windows/downloads/windows-8-sdk) the rebase.exe has been removed. The functionality is still available using EDITBIN /REBASE however, due to **A**ddress **S**pace **L**ayout **R**andomization (ASLR), the rebase has no effect. 
For now, we'll build it without rebasing (further investigation is required).

**Comment by pshipton** [comment]
My understanding is @vsebe has -version working for Windows on a private build. There are about 3 Pull Requests that need to be created, reviewed, and merged before this is available to everyone. Note this is with the bytecode interpreter compiling with no optimization, which affects performance, and this still needs to be addressed afterwards.

```
build/windows-x86_64-normal-server-release/images/jdk/bin/java -version
openjdk version "9-internal"
OpenJDK Runtime Environment (build 9-internal+0-adhoc.AzureADVIOLETASebe.openj9-openjdk-jdk9latest)
Eclipse OpenJ9 VM (build 2.9, JRE 9 Windows 8.1 amd64-64 Compressed References 20171016_000000 (JIT enabled, AOT enabled)
OpenJ9   - ca639b5
OMR      - 38488b78
OpenJDK  - 275ed6548b based on jdk-9+181)
```

**Comment by vsebe** [comment]
Another issue is the forced load of the dbgwrapper for windows platforms only (no dbgwrapper90). We'll work around it and not load it for eclipse openj9 builds.

**Comment by pshipton** [comment]
I think everything is now merged to allow Windows "-version"? @vsebe is this correct?

**Comment by vsebe** [comment]
I had to hack the environment to use the right linker, the msvs not the version packed by cygwin. With this hack `java -version` works.

**Comment by pshipton** [comment]
Thanks. What is also pending are instructions for setting up the build environment.

**Comment by vsebe** [comment]
For instructions please see https://github.com/ibmruntimes/openj9-openjdk-jdk9/issues/18

**Comment by vsebe** [comment]
Closing this issue

**Comment by pshipton** [comment]
Building Windows is working now.

**Comment by aguibert** [comment]
@pshipton are the windows builds published anywhere?  I still only see linux distro out on adoptopenjdk.net :
https://adoptopenjdk.net/releases.html?variant=openjdk9-openj9

**Comment by pshipton** [comment]
@aguibert Adopt is having problems getting the builds working ibmruntimes/openj9-openjdk-jdk9#101
Right now you need to build your own. Windows 8 support was only merged today  ibmruntimes/openj9-openjdk-jdk8#19
